packages/core/src/components/Dropdown/__tests__/driver.js

```diff
@@ -24,6 +24,18 @@ export default class DropdownDriver {
   render() {
     cleanup();
     this.renderResult = render(<Dropdown {...this.props} />);
+    return this.renderResult;
+  }
+
+  async renderWithEffects() {
+    cleanup();
+    await act(async () => {
+      this.renderResult = render(<Dropdown {...this.props} />);
```

Comment:
are you sure you want a new render here? maybe `rerender` would fit better here?

Reply:
I don't have a dom element to send to the createPortal so I guess I can't